extension/zpagesextension/zpagesextension.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package zpagesextension implements an extension that exposes zPages of
```

Comment:
Is there better documentation available for zpages other than https://github.com/census-instrumentation/opencensus-go/tree/master/zpages? I dont' know what it means to be properly instrumented.

Reply:
If it is instrumented with OC the data can be exposed as zPages
 https://opencensus.io/zpages/ otel should preserve this capability.